src/common/auth.js

```diff
@@ -87,6 +87,24 @@ function getAccessToken(callback) {
   );
 }
 
+/**
+ * Calculate the URI where to redirect the user after logging in/out.
+ * Note: won't work if the site is hosted in a subfolder.
+ */
+function getRedirectUri() {
```

Comment:
Wouldn't `window.location.origin` do the same thing as this method?

Reply:
that makes sense, wasn't aware of that function